openstack/containerorchestration/v1/bays/results.go

```diff
@@ -0,0 +1,42 @@
+package bays
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a bay resource.
+func (r commonResult) Extract() (*Bay, error) {
+	var bay *Bay
+	err := r.ExtractInto(&bay)
+	return bay, err
+}
+
+// GetResult represents the result of a get operation.
+type GetResult struct {
+	commonResult
+}
+
+// Represents a Container Orchestration Engine Bay, i.e. a cluster
+type Bay struct {
```

Comment:
It looks like there are several more fields in the unit test example response. Are the missing fields dependent on yet-to-created packages?